### ISSUE TITLE
Elastic Search opinion search clean up punchlist

### ISSUE THREAD

**Issue by mlissner** [freelawproject/courtlistener#3604]
A few quick things we need to do before we can make the opinions live: 

1. The bell icon is kind of a mess. This isn't part of Elastic, but we should spend a few minutes to fix it: 

    ![image](https://github.com/freelawproject/courtlistener/assets/236970/721a7f73-e166-4f02-a04b-e54be3487fe1)

2. For some reason, it's doing "Citation" searches (see the header above, "6,960,345 Opinions cite", and the "0 References to this case").

3. When you do [a citation query](https://www.courtlistener.com/?q=cites%3A(220342%20OR%209441923%20OR%209441924%20OR%209441925)) for the most cited case (Strikland v. Washington), it still says "0 References" instead of saying the right number.

4. The "Cited By" number on the opinion page differs from the number in the search results. Here, cited by shows 76k: 

![image](https://github.com/freelawproject/courtlistener/assets/236970/cd2e73a1-b36a-46ee-8903-9cca881e2e17)

But the query when you click the button (and then enable all the other statuses) has 73k: 

![image](https://github.com/freelawproject/courtlistener/assets/236970/79b7d723-d6fc-4782-b83f-a4bddd4c8135)

5. This query crashes (see: [COURTLISTENER-62V](https://freelawproject.sentry.io/issues/4870029976/)): https://www.courtlistener.com/?order_by=score%20desc&q=cluster_id%3A%20110594&stat_Published=on&type=o

That's my first glance!

**Comment by albertisfu** [comment]
Another issue to consider: Courts are not visible on the advanced search page `/opinion/`

![Screenshot 2024-01-16 at 14 09 23](https://github.com/freelawproject/courtlistener/assets/486004/3da6f982-2618-4786-bd52-4258303d73c0)
